.github/workflows/ci.yml

```diff
@@ -21,13 +21,27 @@ jobs:
       - name: Setup Terraform CLI
         uses: hashicorp/setup-terraform@v2
 
-      - name: Run format check
+      - name: Run Terraform init
+        run: terraform init
+
+      - name: Run Terraform format check
         run: terraform fmt --check
-        continue-on-error: true
```

Comment:
can we continue, but still block the merge?

Reply:
Not with some dedicated option (from what I saw). However, there is a workaround by using `if: always()` or something like this but I didn't like it. The pipeline runs for about 30s so it's not such a big deal to break it early I guess.